frontend/javascripts/test/snapshots/public/test-bundle/test/backend-snapshot-tests/tasks.e2e.js.md

```diff
@@ -56,260 +56,9 @@ Generated by [AVA](https://ava.li).
-        isPublic: false,
-        meshes: [],
-        modified: 0,
-        name: '',
-        organization: 'Organization_X',
-        restrictions: {
-          allowAccess: true,
-          allowDownload: true,
-          allowFinish: true,
-          allowUpdate: true,
-        },
-        settings: {
-          allowedModes: [
-            'orthogonal',
-            'oblique',
-            'flight',
-          ],
-          branchPointsAllowed: true,
-          somaClickingAllowed: true,
-        },
-        state: 'Active',
-        stats: {},
-        tags: [
-          '2012-06-28_Cortex',
-          'skeleton',
-        ],
-        task: {
-          boundingBox: null,
-          created: 1477666728000,
-          creationInfo: null,
-          dataSet: '2012-06-28_Cortex',
-          editPosition: [
-            0,
-            0,
-            0,
-          ],
-          editRotation: [
-            0,
-            0,
-            0,
-          ],
-          formattedHash: '8a5352',
-          id: '581367a82faeb37a008a5352',
-          neededExperience: {
-            domain: 'abc',
-            value: 0,
-          },
-          projectName: 'Test_Project',
-          script: null,
-          status: {
-            active: 1,
-            finished: -1,
-            open: 10,
-          },
-          team: 'team_X1',
-          tracingTime: null,
-          type: {
-            description: 'Check those cells out!',
-            id: '570b9f4c2a7c0e4c008da6ee',
-            recommendedConfiguration: null,
-            settings: {
-              allowedModes: [
-                'orthogonal',
-                'oblique',
-                'flight',
-              ],
-              branchPointsAllowed: true,
-              somaClickingAllowed: true,
-            },
-            summary: 'ek_0563_BipolarCells',
-            teamId: '570b9f4b2a7c0e3b008da6ec',
-            teamName: 'team_X1',
-            tracingType: 'skeleton',
-          },
-        },
-        tracing: {
-          skeleton: '07fd0bd7-f7a6-46d1-aaa8-58a381da4d50',
-          volume: null,
-        },
-        tracingStore: {
-          name: 'localhost',
-          url: 'http://localhost:9000',
-        },
-        tracingTime: null,
-        typ: 'Task',
-        user: {
-          email: 'user_A@scalableminds.com',
-          firstName: 'user_A',
-          id: '570b9f4d2a7c0e4d008da6ef',
-          isAnonymous: false,
-          lastName: 'BoyA',
-          teams: [
-            {
-              id: '570b9f4b2a7c0e3b008da6ec',
-              isTeamManager: true,
-              name: 'team_X1',
-            },
-            {
-              id: '59882b370d889b84020efd3f',
-              isTeamManager: false,
-              name: 'team_X3',
-            },
-            {
-              id: '59882b370d889b84020efd6f',
-              isTeamManager: true,
-              name: 'team_X4',
-            },
-          ],
-        },
-      },
-    ]
+    []
```

Comment:
This also doesn't seem right, or did the PR change something in this regard?

Reply:
I re-ran the snapshot tests on my machine and things look better now :thinking: